src/selectors/getShortcutsModal.js

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+export default function getShortcutsModal(ownProps) {
```

Comment:
hmm, why do we need this?

Reply:
i agree that we don't need it, since it maps props to... props